### PR TITLE
cleanUp changes

### PR DESCRIPTION
**Pull request by galmarko1**

### CODE REVIEW

File: rebalancer/mocking/base.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import (
+	"context"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
+	"github.com/offerm/lnagent/lightning"
+	"google.golang.org/grpc"
+)
+
+type baseMockService struct {
```
Comment (offerm) [inline rebalancer/mocking/base.go:11]: change the file name to be similar to the struct

File: rebalancer/mocking/happyFlowMockService.go [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 	"crypto/sha256"
 	"encoding/hex"
 	"github.com/google/uuid"
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
-	"google.golang.org/grpc"
+	"github.com/offerm/lnagent/lightning"
 )
 
 type mockService struct {
```
Comment (offerm) [inline rebalancer/mocking/happyFlowMockService.go:12]: rename to happy flow

File: rebalancer/mocking/initMockService.go [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 	"github.com/google/uuid"
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
 	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/offerm/lnagent/lightning"
 	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
 type TestInitMockService struct {
```
Comment (offerm) [inline rebalancer/mocking/initMockService.go:13]: name

File: rebalancer/mocking/swapMockService.go [hunk 4]
```diff
@@ -7,12 +7,13 @@ import (
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
 	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/offerm/lnagent/lightning"
 	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
 type TestMockService struct {
```
Comment (offerm) [inline rebalancer/mocking/swapMockService.go:15]: name

File: rebalancer/mocking/base.go (hunk 1 above)
Comment (galmarko1) [reply rebalancer/mocking/base.go:11]: done

File: rebalancer/mocking/happyFlowMockService.go (hunk 2 above)
Comment (galmarko1) [reply rebalancer/mocking/happyFlowMockService.go:12]: done

File: rebalancer/mocking/initMockService.go (hunk 3 above)
Comment (galmarko1) [reply rebalancer/mocking/initMockService.go:13]: done

File: rebalancer/mocking/swapMockService.go (hunk 4 above)
Comment (galmarko1) [reply rebalancer/mocking/swapMockService.go:15]: done

**Review by offerm — APPROVED** [review]

**Pull request merged**